packages/core-contracts/test/utils/Ownable.test.js

```diff
@@ -0,0 +1,196 @@
+const { ethers } = hre;
+const assert = require('assert/strict');
+
+const { ZERO_ADDRESS } = require('../test-libs/test-lib');
+
+describe('Ownable', () => {
+  describe('Constructor / Deployment', () => {
```

Comment:
Pls follow test pattern from other tests.
See `test/ContractUtil.test.js`

We use a before for identifying signers, another for deploying / connecting to contracts, etc.

Reply:
@ajsantander please confirm and explain the reasoning. Retaining the same state across test cases is an anti-pattern that will produce flaky tests and forbid parallelizations of test running.

Also mocha has a context (`this`) and it's encouraged to use it when setting up test cases vs having global variables like in the `test/ContractUtil.test.js` test module.

Reply:
Again, this is the pattern we are following. I am familiar with your reasoning, I am just asking you to adapt to our current practices.